Riot/Modules/Room/Settings/RoomSettingsViewController.m

```diff
@@ -3832,23 +3832,29 @@ - (void)toggleDirectoryVisibility:(UISwitch*)theSwitch
 
 - (void)showRoomAccessFlow
 {
-    MXRoom *room = [self.mainSession roomWithRoomId:self.roomId];
-    
-    if (room) {
-        roomAccessPresenter = [[RoomAccessCoordinatorBridgePresenter alloc] initWithRoom:room];
-        roomAccessPresenter.delegate = self;
-        [roomAccessPresenter presentFrom:self animated:YES];
+    if (@available(iOS 14.0, *))
```

Comment:
It is surprising that this check was not previously needed, given that `RoomAccessCoordinatorBridgePresenter` is iOS14 only.

Reply:
It was definitively a mistake.